client/app/hearings/components/assignHearings/AssignHearingsTable.jsx

```diff
@@ -1,29 +1,271 @@
+    if (_.isNull(suggestedLocation) || _.isUndefined(suggestedLocation)) {
+      return null;
+    }
+
+    const { city, state } = suggestedLocation;
+    const formattedFacilityType = suggestedLocation.formatted_facility_type;
+
+    return `${city}, ${state} ${formattedFacilityType}`;
+  }
+
+  getCurrentPageNumberFromUrl = () => {
+    const queryParams = getQueryParams(window.location.search);
+
+    return Number(queryParams[QUEUE_CONFIG.PAGE_NUMBER_REQUEST_PARAM]) || 1;
+  }
+
+  /*
+   * Gets the list of columns to populate the QueueTable with.
+   */
+  getColumns = () => {
+    const { selectedRegionalOffice, selectedHearingDay } = this.props;
+
+    const { colsFromApi } = this.state;
+
+    if (_.isNil(selectedHearingDay)) {
+      return [];
+    }
+
+    const columns = [
+      {
+        header: '',
+        align: 'left',
+        // Since this column isn't tied to anything in the input row, _value will
+        // always be undefined.
+        valueFunction: (_value, rowId) => <span>{rowId + this.state.rowOffset}.</span>
+      },
+      {
+        name: 'caseDetails',
+        header: 'Case Details',
+        align: 'left',
+        valueFunction: (row) => (
+          <LinkToAppeal
+            appealExternalId={row.externalAppealId}
+            hearingDay={selectedHearingDay}
+            regionalOffice={selectedRegionalOffice}
+          >
+            <CaseDetailsInformation appeal={row.appeal} />
+          </LinkToAppeal>
+        )
+      },
+      {
+        name: 'type',
+        header: 'Type(s)',
+        align: 'left',
+        valueFunction: (row) => renderAppealType({
+          caseType: row.appeal.caseType,
+          isAdvancedOnDocket: row.appeal.isAdvancedOnDocket
+        })
+      },
+      {
+        name: 'docketNum',
+        header: 'Docket Number',
+        align: 'left',
+        valueFunction: (row) => (
+          <AppealDocketTag appeal={row.appeal} />
+        )
+      },
+      {
+        name: 'suggestedLocation',
+        header: 'Suggested Location',
+        align: 'left',
+        columnName: 'Suggested Location',
+        valueFunction: (row) => (
+          <SuggestedHearingLocation
+            suggestedLocation={row.suggestedHearingLocation}
+            format={this.formatSuggestedHearingLocation}
+          />
+        ),
+        label: 'Filter by location',
+        filterValueTransform: this.formatSuggestedHearingLocation,
+        enableFilter: true,
+        anyFiltersAreSet: true,
+        filterOptions: colsFromApi && colsFromApi.find((col) => col.name === 'suggestedLocation').filter_options
+      },
+      {
+        name: 'powerOfAttorneyName',
+        header: 'Power of Attorney (POA)',
+        columnName: 'Power of Attorney',
+        valueFunction: (row) => (
+          <PowerOfAttorneyDetail
+            key={`poa-${row.externalAppealId}-detail`}
+            appealId={row.externalAppealId}
+            displayNameOnly
+          />
+        ),
```

Comment:
```suggestion
        ),
        label: 'Power of Attorney',
```
I think we need to add a label here for accessibility (and to avoid sporadic Sniffybara `Buttons must have discernible text` errors in feature tests).

Reply:
Done!